dpl-docs/views/layout.dt

```diff
@@ -113,6 +113,8 @@ html(lang='en-US')
         h1 <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[0 .. $-24]}</code> #{title[$-24 ..$]}
       - else if (hidx > 0 && title[0 .. hidx] != "Module" && info.mod)
         h1 #{title[0 .. hidx]} <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[hidx+1 .. $]}</code>
+      - else if (hidx > 0 && title[0 .. hidx] == "Module" && info.mod)
+        h1 Module <code>#{info.mod.qualifiedName}</code>
```

Comment:
Nope, diet templates are pure HTML templates, so they would not understand DDOC's macros.
But what do you want to achieve ? Linking ?